editor/src/components/common/shared-strategies/convert-to-flex-strategy.ts

```diff
@@ -245,12 +247,81 @@ function ifElementIsFragmentLikeFirstConvertItToFrame(
   return []
 }
 
+const NEAR_CENTER_LINE_TOLERANCE = 0.2
```

Comment:
Very small nitpick, but I think it makes this a tiny bit clearer:
```suggestion
const NEAR_CENTER_LINE_TOLERANCE_MULTIPLIER = 0.2
```

Reply:
renamed it in https://github.com/concrete-utopia/utopia/pull/4399/commits/3f6ef2de0dbe9df20f8e31d5e91f0b87623b8bfd